compiler/aarch64/codegen/OMRMemoryReference.cpp

```diff
@@ -23,7 +23,9 @@
 
 #include <stddef.h>
 #include <stdint.h>
+#include "codegen/ARM64Instruction.hpp"
 #include "codegen/CodeGenerator.hpp"
+#include "codegen/Instruction.hpp"
```

Comment:
Do you need to include this as well as `ARMInstruction.hpp` ?  ARMInstruction is an extension of Instruction so this must be included already.  Please remove.